model/registry/cloudfoundry.yaml

```
@@ -0,0 +1,140 @@
+groups:
+  - id: registry.cloudfoundry
+    prefix: cloudfoundry
+    type: resource
```

If we have a mix of resource/signal attributes in the registry, I think we need to mark these as an attribute group.

```suggestion
    type: attribute_group
```

We have an issue open that is tracking changes we need to make in the tooling to support a mix like this https://github.com/open-telemetry/build-tools/issues/242. A PR was merged recently but I'm not sure if that alone solves everything. Please @lmolkova @AlexanderWert could you confirm if that's the case?

---

Currently, all attributes are proposed to be resource attributes. I keep this suggestion in mind for the discussion about the `source.type`.

---

I made that change in the last commit after merging the current main branch into this PR.